scripts/api/data/licenses/licenseOGL-UK-3.0.json

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "OGL UK 3.0",
+  "uri": "https://www.nationalarchives.gov.uk/doc/open-government-licence/version/3",
+  "shortDescription": "Open Government Licence v3.0.",
```

Comment:
Just a heads up to @jggautier that I used the British spelling of "license" with two c's, which differs from his zip at #9403 which I used as a starting point (thanks!).

Reply:
Well, let's see what we settle on for this PR. If we merge the British spelling, yes, we should probably change Harvard Dataverse to match.

Another thought is that we have `"languageCode": "en"`. Maybe in the future we could have en-US vs en-UK or whatever.